spec/features/posts_feature_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+feature 'posts' do
+  context 'no posts have been added' do
+    scenario 'should display a prompt to add a post' do
+      visit '/'
+      expect(page).to have_content 'No posts yet!'
+      expect(page).to have_link 'New Post'
+    end
+  end
+end
+
+feature 'Creating posts' do
```

Comment:
There's quite a lot of repetition across these tests. Consider how you could refactor a helper method.